packages/material-ui/src/transitions/utils.js

```diff
@@ -1,8 +1,6 @@
 export const reflow = node => node.scrollTop;
 
-export function getTransitionProps(props, options) {
-  const { timeout, style = {} } = props;
-
+export function getTransitionProps({ timeout, style = {} }, options) {
```

Comment:
We try to not destructure early. I agree with @oliviertassinari here that this makes the code actually less readable.